core/controllers/editor.py

```diff
@@ -303,6 +304,8 @@ def _get_exploration_data(self, exploration_id, version=None):
             'tags': exploration.tags,
             'title': exploration.title,
             'version': exploration.version,
+            'is_draft_version_valid': exp_services.is_draft_version_valid(
```

Comment:
I keep reading 'is_draft_version_valid' as meaning 'is this draft version (i.e. changelist) one that is valid' rather than 'is the version of this draft valid'. I would suggest calling it 'is_draft_valid_version' instead.